ape_etherscan/explorer.py

```diff
@@ -1,14 +1,32 @@
 from ape.api import ExplorerAPI
+from ape.types import ABI, ContractType
+from typing import List
+import requests
+import json
 
 ETHERSCAN_URI = (
     lambda n: (f"https://{n}.etherscan.io/" if n != "mainnet" else "https://etherscan.io/")
     + "{0}/{1}/"
 )
 
+ETHERSCAN_API_URI = lambda n: (
```

Comment:
It does make much sense to make this an anonymous function because it is not really anonymous. `lambda` should only really be used when passing a function as an argument to something.

```suggestion
def get_etherscan_uri(network_name):
```

Reply:
had to refactor a little but this is working now